### DISCUSSION TITLE
Ice Accumulation (Missing from Grids)

### DISCUSSION THREAD

**Discussion by grfritz** [weather-gov/api#376] [Q&A]
While exploring the precipitation-associated forecast grids available in the API web service I noticed that Ice Accumulation is consistently missing (i.e. "values": []) from wfo Seattle as noted at:

https://api.weather.gov/gridpoints/SEW/31,80

I want to check whether this is a bug, or, perhaps whether some wfo simply do not provide this grid.  If so, I will need to develop my app accordingly  (indicating it is an error, or, simply not provided).

Thanks

**Comment by StephenClouse** [comment]
This is actually what we are receiving from the office. Not a bug.
